distributed/shuffle/tests/test_shuffle.py

```diff
@@ -74,10 +74,10 @@ async def clean_scheduler(
     assert not extension.heartbeats
 
 
-@pytest.mark.skipif(
-    pa is not None,
-    reason="We don't have a CI job that is installing a very old pyarrow version",
-)
+# @pytest.mark.skipif(
+#     pa is not None,
+#     reason="We don't have a CI job that is installing a very old pyarrow version",
+# )
```

Comment:
```suggestion
```

Reply:
It's clear to me why this change was made to this test. My impression is that what's currently on `main` is still want we want. I've reverted back to what's on `main` (confirmed the test passes locally when `pyarrow` isn't installed). Happy to submit a follow-up PR if needed